src/overlays/actors/ovl_Obj_Ocarinalift/z_obj_ocarinalift.c

```diff
@@ -28,56 +43,228 @@ const ActorInit Obj_Ocarinalift_InitVars = {
     (ActorFunc)ObjOcarinalift_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80AC9D70[] = {
-    ICHAIN_U8(targetMode, 2, ICHAIN_CONTINUE),
-    ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
-    ICHAIN_F32(uncullZoneScale, 200, ICHAIN_CONTINUE),
-    ICHAIN_F32(uncullZoneDownward, 300, ICHAIN_CONTINUE),
+static InitChainEntry sInitChain[] = {
+    ICHAIN_U8(targetMode, 2, ICHAIN_CONTINUE),         ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 200, ICHAIN_CONTINUE), ICHAIN_F32(uncullZoneDownward, 300, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+void func_80AC94C0(ObjOcarinalift* this, s32 arg1) {
+    Math_Vec3s_ToVec3f(&this->dyna.actor.world.pos, &this->unk170[arg1]);
+}
+
+void ObjOcarinalift_Init(Actor* thisx, PlayState* play) {
+    Path* path;
+    ObjOcarinalift* this = THIS;
+
+    Actor_ProcessInitChain(&this->dyna.actor, sInitChain);
+    this->dyna.actor.shape.rot.x = 0;
+    this->dyna.actor.world.rot.x = 0;
+    this->dyna.actor.shape.rot.z = 0;
+    this->dyna.actor.world.rot.z = 0;
+    DynaPolyActor_Init(&this->dyna, 1);
+    DynaPolyActor_LoadMesh(play, &this->dyna, &object_raillift_Colheader_0048D0);
+    this->unk160 = this->dyna.actor.home.rot.z * 0.1f;
+    if (1) {}
```

Comment:
Can this be eliminated by thisx use too?

Reply:
yes, it worked! thank you.